src/multi_vector_simulator/cli.py

```diff
@@ -151,5 +160,82 @@ def main(**kwargs):
     return 1
 
 
+def report(pdf=None, path_simulation_output_json=None, path_pdf_report=None):
+
+    """Display the report of a MVS simulation
+
```

Comment:
Add a line on how to execute the function now, I only see which arguments I could do when execute it

Reply:
Done in b7df6e5df85cce4c0dfb34c1a9e123920816a7bf